person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initialize(age, name = 'unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  def can_use_services?
+    @parent_permission || is_of_age?
```

Comment:
- [ ] the name of your private method is `of_age` not `is_of_age` kindly fix that

Reply:
Thank you.
Done